keyboards/work_louder/loop/keymaps/PeterFalken/config.h

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 PeterFalken  (@PeterFalken)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+// Fix warning - "Attached USB accessory uses too much power."
+#ifndef USB_MAX_POWER_CONSUMPTION
+    #define USB_MAX_POWER_CONSUMPTION 100
+#endif
+
+// Fix unresponsive on wake from sleep
+#ifndef USB_SUSPEND_WAKEUP_DELAY
+    #define USB_SUSPEND_WAKEUP_DELAY 200
+#endif
+
+#undef RGB_MATRIX_DEFAULT_MODE
+#define RGB_MATRIX_DEFAULT_MODE 0
+
+// Space optimizations - 01
+#undef LOCKING_SUPPORT_ENABLE
+#undef LOCKING_RESYNC_ENABLE
+
+// Space optimizations - 02
+#undef RGBLIGHT_ANIMATIONS
```

Comment:
```suggestion
#undef RGBLIGHT_EFFECT_BREATHING
#undef RGBLIGHT_EFFECT_RAINBOW_MOOD
#undef RGBLIGHT_EFFECT_RAINBOW_SWIRL
#undef RGBLIGHT_EFFECT_SNAKE
#undef RGBLIGHT_EFFECT_KNIGHT
#undef RGBLIGHT_EFFECT_CHRISTMAS
#undef RGBLIGHT_EFFECT_STATIC_GRADIENT
#undef RGBLIGHT_EFFECT_RGB_TEST
#undef RGBLIGHT_EFFECT_TWINKLE
```

Reply:
No extra space was gained from undefining the RGBLIGHT_EFFECT_*